psi4/driver/frac.py

```diff
@@ -27,29 +27,89 @@
 #
 
 from __future__ import absolute_import
+from __future__ import division
```

Comment:
Maybe also `print_function` to avoid py2 devs screwing things up, if you happen to modify this file again at some point.  No worry if not.